src/transport.rs

```diff
@@ -57,15 +57,13 @@ impl KKTransport {
     /// as a responder with our single private key and a set of possible public key for them.
     /// This is used by servers to identify the origin of the message.
     pub fn accept(
-        listener: &TcpListener,
+        mut connection: TcpStream,
```

Comment:
You need to change the doc comment as well.
```suggestion
    /// Perform the noise KK handshake as a responder with our single private key and a set of possible
    /// public keys for them.
    /// This is used by servers to identify the origin of the message.
    pub fn accept(
        mut connection: TcpStream,
```